### ISSUE TITLE
Error with fbad_read_adset

### ISSUE THREAD

**Issue by negasonic19** [daroczig/fbRads-transition#1]
I have been able to pull data from facebook using fbRads library. For ex :  
`> fbad_read_adset(mac, '6069175388378', fields = c("name"))`  
where mac is the object created when i validated my token using 'fbad_init' for an account id. 
However when i use the 'targeting' to pull the targeting of the adset i am getting the following error:
`> fbad_read_adset(mac, '6069175388378', fields = c("name","targeting"))`
Error in (function (..., row.names = NULL, check.rows = FALSE, check.names = TRUE,  : 
  arguments imply differing number of rows: 1, 7, 2

Basically i want the list of all the targeting for all adsets in my account.

**Comment by daroczig** [comment]
Hey, this repo is live for only a short period of time (transition), so please use https://github.com/cardcorp/fbRads for reporting issues in the future. But I've just pushed 3fdb2b9a866fbed8b2913c255d9b42f0965b932f providing a new arg to return `list` instead of trying to coerce into `data.frame` -- I hope this resolves your issue
